sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/EnableSdaPlatform.java

```diff
@@ -38,8 +39,9 @@
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
-@EnableSdaDocs
 @EnableSdaRestGuide
+@EnableSdaWebSecurity
+@EnableSdaDocs
```

Comment:
I would've preferred `EnableSdaOpenApiDocs` or similar but that's not part of the review / changes.